ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -1426,38 +1441,6 @@ impl<Register: Send + Sync + 'static> TraceBuilder<Register> {
                             // started outlining. We are done and can continue processing
                             // blocks normally.
                             self.outline_target_blk = None;
-                            // If we started outlining due to a `yk_idempotent` annotation we must
-                            // now put the dynamically recorded return value into the (already
-                            // emitted) call instruction for the trace optimiser to pick up later.
-                            //
-                            // Because we haven't emitted any instructions (only outlined) since
-                            // emitting the call to the idempotent function, the call we need to
-                            // modify is still the last instruction in the JIT module.
-                            let last_iidx = self.jit_mod.last_inst_idx();
-                            // It's essential we use `inst_raw` here, since a call to an idempotent function is
-                            // followed by an call to `__yk_idempotent_promote_*()` which the trace
-                            // builder will convert into a [Inst::Copy] to the first call. If
-                            // we were to decopy the instruction we'd consume from the promote
-                            // buffer twice!
-                            let inst = self.jit_mod.inst_raw(last_iidx);
```

Comment:
I am so happy the `inst_raw` call has gone! FWIW, if you need this again, you *probably* really want `inst_nocopy`.